### ISSUE TITLE
output messages are writing inconsistently to stdout & stderr

### ISSUE THREAD

**Issue by davidk81** [infostellarinc/stellarcli#126]
logging is not consistent across the application; some logs write to stdout, some to stderr.
although a central logger is now used, some calls are redirected to "log" library which logs to stderr by default.

note that changing logging behavior can potentially break integration for some customers that rely on log parsing.

**Comment by chelliwell** [comment]
> note that changing logging behavior can potentially break integration for some customers that rely on log parsing.

Yep, I can see that! It's perhaps safest as-is if you want to avoid dropping breaking changes on anyone. Your call :)
I actually now hook both stderr and stdout and then...... note!! ..... assume that, regardless of the pipe, an error message will contain "error", and a non-error message won't. I then log to our own logger library, setting the log level as appropriate.

**Comment by chelliwell** [comment]
Hi, I wondered if you'd been able to look at this one yet. One headache I have is with 'list-plans', which prints
2021/05/13 16:20:57 API endpoint: api.stellarstation.com:443
to _stderr_, so my app generates an erroneous error message.

**Comment by chelliwell** [comment]
> Hi, I wondered if you'd been able to look at this one yet. One headache I have is with 'list-plans', which prints
> 2021/05/13 16:20:57 API endpoint: api.stellarstation.com:443
> to _stderr_, so my app generates an erroneous error message.

And yet, I'm parsing stderr for the JSON plans..... so maybe the best way for consistency would indeed be to make sure that error messages contain 'errors?
